packages/gui/src/components/vcs/VCCard.tsx

```diff
@@ -55,6 +54,7 @@ export default function VCCard(props: { vcRecord: any; isDetail?: boolean; proof
   const [pendingRevoke, setPendingRevoke] = useLocalStorage<any>('verifiable-credentials-pending-revoke', {});
   const { data: fingerprint } = useGetLoggedInFingerprintQuery();
   const [getTransactionAsync] = useGetTransactionAsyncMutation();
+  const isVCLocal = React.useMemo(() => !!vcRecord.isLocal, [vcRecord.isLocal]);
```

Comment:
Why is this using useMemo? Also, shouldn't this be checking if vcRecord is undefined? We check `vcRecord?.<property>` elsewhere.

Reply:
Maybe `isLocal` should be a VCCard prop instead of attaching the property to vcRecord.

Reply:
Fixed.